packages/customer/src/migrations/Migration20240124154000.ts

```diff
@@ -23,7 +23,7 @@ export class Migration20240124154000 extends Migration {
       'create index if not exists "IDX_customer_address_customer_id" on "customer_address" ("customer_id");'
     )
     this.addSql(
-      'create unique index "IDX_customer_address_unqiue_customer_billing" on "customer_address" ("customer_id") where "is_default_billing" = true;'
+      'create unique index "IDX_customer_address_unique_customer_billing" on "customer_address" ("customer_id") where "is_default_billing" = true;'
```

Comment:
@srindom by trully running the migrations as part of the integration tests, I encounter this one 😂 until now, the test-utils wasn't running the migration at all. It might help forcing everybody to test migrations as well

cc @olivermrbl @carlos-r-l-rodrigues

Reply:
It means that until now, even if the integration tests was green, it could be not working in truth cause of the discrepancy between the definition and the migrations

Reply:
fixed in another pr